Userland/DevTools/HackStudio/Git/GitWidget.cpp

```diff
@@ -133,13 +134,18 @@ void GitWidget::unstage_file(const LexicalPath& file)
 
 void GitWidget::commit()
```

Comment:
A bit of a nitpick but the logic here could be nicer if we flipped the if/else logic and did early return.